cli_app/main.rb

```diff
@@ -0,0 +1,27 @@
+require './cli_codeable.rb'
+
+begin
+  while true == true
+  puts logo ="
+    ____          _            _     _      
+   / ___|___   __| | ___  __ _| |__ | | ___ 
+  | |   / _ \\ / _` |/ _ \\/ _` | '_ \\| |/ _ \\
+  | |__| (_) | (_| |  __/ (_| | |_) | |  __/
+   \\____\\___/ \\__,_|\\___|\\__,_|_.__/|_|\\___|                                  
+  ".blue
+  1.times {puts ""}
+
+  puts "Type: 't title of task' to add a new todo list item.".cyan
+  puts "Type: 't' to show your task list.".cyan
+  puts "Type: 't -d [task number]' to complete a task and clean.".cyan
+  puts "Type: 'quit' or 'q' to quit.".cyan
+
+  puts ""
+  show
+  menu
+  all
+  end
+
+rescue Interrupt => e
```

Comment:
It's the reference of the exception!

When Ruby finds an Interrupt error/exception, it will run the code just below (line 26 in this case) and you can access extra information about the error inside the `e` variable.

You can name it whatever you want but e as a short version of error usually works well enough, but if you're having multiple rescue blocks for different exceptions, having more descriptive names is probably a good idea.

Reply:
Thanks, Arturo